src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -130,6 +130,13 @@
                 .desc("When present this flag indicates that external content should be exported.")
                 .required(false).build());
 
+        // Retrieve inbound references
+        configOptions.addOption(Option.builder("i")
```

Comment:
I can see value in making the use of various options clear with respect to their applicability to either import or export or both.
That could be done in the CLI help output and/or with WARNING log output with unused arguments.